src/nomad_catalysis/schema_packages/catalysis.py

```diff
@@ -394,3 +412,1107 @@ def normalize(self, archive, logger):
+                    'Fritz-Haber-Institut Berlin / ISC',
+                    'TU Berlin, BASCat',
+                    'HZB',
+                    'CATLAB',
+                ]
+            ),
+        ),
+    )
+
+    experimenter = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        The person that performed or started the measurement.
+        """,
+        a_eln=dict(component='EnumEditQuantity'),
+    )
+
+
+class CatalyticReactionData(PlotSection, MeasurementResult, ArchiveSection):
+    temperature = Quantity(
+        type=np.float64,
+        shape=['*'],
+        unit='°C',
+        a_eln=ELNAnnotation(component='NumberEditQuantity'),
+    )
+
+    pressure = Quantity(
+        type=np.float64,
+        shape=['*'],
+        unit='bar',
+        a_eln=ELNAnnotation(component='NumberEditQuantity'),
+    )
+
+    runs = Quantity(
+        type=np.float64,
+        shape=['*'],
+        a_eln=ELNAnnotation(component='NumberEditQuantity'),
+    )
+    time_on_stream = Quantity(
+        type=np.float64,
+        shape=['*'],
+        unit='hour',
+        a_eln=ELNAnnotation(component='NumberEditQuantity'),
+    )
+
+    c_balance = Quantity(
+        type=np.dtype(np.float64),
+        shape=['*'],
+        a_eln=ELNAnnotation(component='NumberEditQuantity'),
+    )
+
+    reactants_conversions = SubSection(section_def=ReactantData, repeats=True)
+    rates = SubSection(section_def=RatesData, repeats=True)
+
+    products = SubSection(section_def=ProductData, repeats=True)
+
+    def normalize(self, archive, logger):
+        if self.products is not None:
+            for product in self.products:
+                if product.pure_component is None or product.pure_component == []:
+                    product.normalize(archive, logger)
+
+
+class CatalyticReaction(CatalyticReaction_core, PlotSection, Schema):
+    m_def = Section(
+        label='Catalytic Reaction (filled manual/ by json directly)',
+        a_eln=ELNAnnotation(
+            properties=dict(
+                order=[
+                    'name',
+                    'data_file',
+                    'reaction_name',
+                    'reaction_class',
+                    'experimenter',
+                    'location',
+                    'experiment_handbook',
+                ]
+            )
+        ),
+        categories=[CatalysisElnCategory],
+    )
+
+    reactor_setup = SubSection(section_def=ReactorSetup)
+    reactor_filling = SubSection(section_def=ReactorFilling)
+
+    pretreatment = SubSection(
+        section_def=ReactionConditions_data, a_eln=ELNAnnotation(label='pretreatment')
+    )
+    reaction_conditions = SubSection(
+        section_def=ReactionConditions_data,
+        a_eln=ELNAnnotation(label='reaction conditions'),
+    )
+    results = Measurement.results.m_copy()
+    results.section_def = CatalyticReactionData
+    results.a_eln = ELNAnnotation(label='reaction results')
```

Comment:
@hampusnasstrom: do you know why this line should not work? I was seriously considering the alternative appoach I commented just below, but I understand this is not the approved way to overwrite the results subsection, right?

Reply:
I'm quite confused about this myself. We had a lot of issues with the `m_copy` so we actually end up overwriting the subsections a lot nowadays. You should always make sure that the `section_def` is either the same or inherits from the previous one though!

Reply:
ok, good to know. Thanks!